### PR TITLE
UI : Added tags at schema Level

### LINKED ISSUE #10507: Add / Edit dataabaseSchema level tags for databases
UI: need to have a column for tags for schema level on the databaseSchema under database.

ref for the backend PR: https://github.com/open-metadata/OpenMetadata/pull/10381

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Issue: Closes #10507
- Added tags at schema Level

#
### Type of change :

- [x] New feature

#
### Frontend Preview (Screenshots) :
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Tags.spec.js [hunk 1]
```diff
@@ -155,6 +156,75 @@ describe('Tags page should work', () => {
     addNewTagToEntity(entity, `${NEW_TAG_CATEGORY.name}.${NEW_TAG.name}`);
   });
 
+  it('Add tag at DatabaseSchema level should work', () => {
+    const entity = SEARCH_ENTITY_TABLE.table_2;
+    const term = `${NEW_TAG_CATEGORY.name}.${NEW_TAG.name}`;
+    const term2 = 'PersonalData.Personal';
+    visitEntityDetailsPage(entity.term, entity.serviceName, entity.entity);
+
+    cy.get(':nth-child(3) > .link-title')
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Tags.spec.js:165]: Choose a batter selector or use test id instead

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Tags.spec.js [hunk 2]
```diff
@@ -155,6 +156,75 @@ describe('Tags page should work', () => {
...
+
+    cy.get('[data-testid="submit-test"]').should('be.visible').click();
+
+    // Accept the tag suggestion which is created
+    cy.get('.ant-btn-compact-first-item')
+      .should('be.visible')
+      .contains('Accept Suggestion')
+      .click();
+
+    cy.wait(500);
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Tags.spec.js:218]: Instead wait on API calls

File: openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx [hunk 3]
```diff
@@ -470,6 +483,40 @@ const DatabaseSchemaPage: FunctionComponent = () => {
...
+          })
+          .catch((err: AxiosError) => {
+            showErrorToast(
+              err,
+              jsonData['api-error-messages']['update-databaseSchema-error']
+            );
+            reject();
+          });
+      });
+    }
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx:517]:
```suggestion
try {
        const res = await saveUpdatedDatabaseSchemaData(updatedData as DatabaseSchema)

            if (res) {
              setDatabaseSchema(res);
              setTags(getTagsWithoutTier(res.tags || []));
              setTier(getTierTags(res.tags ?? []));

              getEntityFeedCount();
    } catch (err) {
    showErrorToast(
              err,
             t('server.api-error)
            );
    }
```

**Comment by Ashish8689** [comment]
@chirag-madlani  changes updated!

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**